### PR TITLE
feat(server): design blog comment, replies and like button

### PR DESCRIPTION
**Pull request by nihalanisumit**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [X] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [X] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:
Issue Number: 
#78 

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by nihalanisumit** [comment]
Screenshot from 2019-08-30 05-54-56

**Comment by nihalanisumit** [comment]
Screenshot from 2019-08-30 06-12-03

File: client/src/components/Comment/Comment.module.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.container{
+    padding: 20px 20vw;
+    color: white;
+}
```
Comment (piyushagwl007) [inline client/src/components/Comment/Comment.module.css]: new line

File: client/src/components/Comment/index.module.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.color{
+    color: white !important;
+}
+
+.replyLink{
+    color: orange !important;
+}
```
Comment (piyushagwl007) [inline client/src/components/Comment/index.module.css]: new line?

**Review by piyushagwl007 — COMMENTED** [review]
Let's discuss this P.R. f2f because some of the things are not clear to me

**Comment by nihalanisumit** [comment]
Screenshot from 2019-08-30 16-34-40

**Review by piyushagwl007 — APPROVED** [review]
LGTM

**Pull request merged**